### PR TITLE
Update UG, DG and project portfolio

### LINKED ISSUE #107: [PE-D][Tester B] Wrong link to download jar file in UG
Clicking on the link under Quick Start in UG links me to the v1.2 release instead of the v1.3 release

-------------
Labels: `type.DocumentationBug` `severity.High`
original: SimHanKiong/ped#1

### LINKED ISSUE #108: [PE-D][Tester C] Typo in UG
"explaining" is misspelled as "explaning" under the 'Viewing help' section.

explaning

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: VatsalVitosh/ped#1

### LINKED ISSUE #111: [PE-D][Tester A] Incomplete text display
Full address cannot be viewed if length of address exceeds one line, even when the column on the left has been minimized. 

**See person at index 7**

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: JiaXinEu/ped#1

### LINKED ISSUE #112: [PE-D][Tester B] Adding duplicate people by changing case of name
Adding a duplicate person in lower case is allowed

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: SimHanKiong/ped#3

### LINKED ISSUE #114: [PE-D][Tester B] Adding duplicate people with whitespace in name
Able to add a duplicate person by adding spaces between the names.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: SimHanKiong/ped#4

### LINKED ISSUE #127: [PE-D][Tester E] Previous Addressbook folder
From the FAQ, Addressbook is mentioned although the name of the app is EffiTracker. This can cause confusion among users.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: jinhanfromNUS/ped#3

### LINKED ISSUE #137: [PE-D][Tester E] Adding is allowed when having the same attributes (with name being capitalised)
Person with same name (different capitalization) is allowed, even when all other field is identical. 

add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 d/Business Development t/friends t/owesMoney eff/80
add n/JOHN DOE p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 d/Business Development t/friends t/owesMoney eff/80

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: jinhanfromNUS/ped#6

### LINKED ISSUE #148: [PE-D][Tester E] Invalid email format is accepted 2
Email without com/edu is still accepted

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: jinhanfromNUS/ped#9

### LINKED ISSUE #159: [PE-D][Tester E] Task marking does not take capitalisation into account
When a person has been assigned task 1. `mark task/Task 1 o/1` is unable to mark the task, unless `Task 1 is changed to task 1`.

Personally, capitalisation can be ignored for marking.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: jinhanfromNUS/ped#14

### PR DESCRIPTION
**Pull request by yiwern5**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -90,11 +90,70 @@ outdated systems, EffiTrack offers a centralized platform for easy and accurate
...
+
+* `KEYWORD`: The keyword of the query.
+    * `KEYWORD` is case-insensitive.
+
+* `THRESHOLD`: The threshold of efficiency score of the employee.
+    * Takes in integer values in the range 0 to 100.
+
+* `COMMENT`: The comment about the employee.
+    * Only alphanumeric comments with spaces is allowed.
+    * Special characters are not allowed in comments.
```
Comment (laybach) [inline docs/UserGuide.md]: special characters are allowed in comments too

**Review by laybach — CHANGES_REQUESTED** [review]
minor error for comment part

File: docs/UserGuide.md (hunk 1 above)
Comment (yiwern5) [reply docs/UserGuide.md:150]: no u used name validation regex

**Review by laybach — APPROVED** [review]
LGTM

**Pull request merged**